packages/hooks/src/test/index.test.js

```diff
@@ -620,3 +628,30 @@ test( 'Test `this` context via composition', () => {
 	const testObject2 = {};
 	Object.assign( testObject2, createHooks() );
 } );
+
+// Test adding a hook triggers a hookAdded action.
```

Comment:
Those comments for tests are obsolete as they repeat the test description. Here and in the line 646.

Reply:
ok, fine to remove.